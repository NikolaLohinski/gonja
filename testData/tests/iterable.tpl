{{ 'gonja' is iterable }}
{{ ['gonja'] is iterable }}
{{ {"key": "value"} is iterable }}
{{ 3 is iterable }}
{{ true is iterable }}