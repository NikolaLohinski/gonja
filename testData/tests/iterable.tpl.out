True
True
True
False
False